src/Microsoft.AspNetCore.Sockets/WebSockets.cs

```diff
@@ -11,11 +11,13 @@ public class WebSockets : IHttpTransport
     {
         private readonly HttpChannel _channel;
         private readonly Connection _connection;
+        private readonly WebSocketMessageType _messageType;
 
-        public WebSockets(Connection connection)
+        public WebSockets(Connection connection, Format format)
         {
             _connection = connection;
             _channel = (HttpChannel)connection.Channel;
+            _messageType = format == Format.Binary ? WebSocketMessageType.Binary : WebSocketMessageType.Text;
```

Comment:
Yes. On the way in we consume the stream. I actually talked to @tratcher about this and he said that mode actually does not matter as much - we could always just use binary. Apparently setting message type to .Text just introduces some additional checks on the data being sent (and what receive callback/overload on the client side is invoked). So maybe we can just drop this.